WrightTools/data/_solis.py

```diff
@@ -0,0 +1,110 @@
+"""Andor."""
+
+
+# --- import --------------------------------------------------------------------------------------
+
+
+import os
+import time
+
+import numpy as np
+
+from ._data import Axis, Channel, Data
+from .. import exceptions as wt_exceptions
+from ..kit import _timestamp as timestamp
+
+
+# --- define --------------------------------------------------------------------------------------
+
+
+__all__ = ["from_Solis"]
+
+
+# --- from function -------------------------------------------------------------------------------
+
+
+def from_Solis(filepath, name=None, parent=None, verbose=True):
+    """Create a data object from Andor Solis software (ascii exports).
+
+    Parameters
+    ----------
+    filepath : string, list of strings, or array of strings
+        Path to .txt file.
+    name : string (optional)
+        Name to give to the created data object. If None, filename is used.
```

Comment:
Does file metadata provide a name? if so, that is preferred to the file name itself

Reply:
I took a look, and doesn't seem like there is a name in the footer

guess it makes sense, as I don't imagine that the software requires users to specify a name at acquisition time